migrations/2025-04-12-090344_update-mod_ban-instance_id/up.sql

```diff
@@ -0,0 +1,13 @@
+UPDATE
+    mod_ban
+SET
+    instance_id = person.instance_id
+FROM
+    person
+WHERE
+    mod_ban.instance_id IS NULL
+    AND mod_ban.mod_person_id = person.id;
+
+ALTER TABLE mod_ban
+    ALTER COLUMN instance_id SET NOT NULL;
+
```

Comment:
Might as well just add all these lines to the previous migration, since it hasn't been included in any release yet.